pages/configuration.tsx

```diff
@@ -1,22 +1,41 @@
 import type { NextPage } from "next";
 import { useEffect, useState } from "react";
+import { AppBridgeState } from "@saleor/app-bridge";
 
+
+import { SALEOR_DOMAIN_HEADER } from "../constants";
 import { useFetchTwelveOrdersQuery } from "../generated/graphql";
 import useApp from "../hooks/useApp";
 
 const Configuration: NextPage = () => {
-  const [{ data }] = useFetchTwelveOrdersQuery();
   const app = useApp();
-  const [domain, setDomain] = useState("");
 
-  useEffect(() => setDomain(app?.getState()!.domain || ""), [app]);
+  const [appState, setAppState] = useState<AppBridgeState>();
+  const [configuration, setConfiguration] = useState(null);
+
+  const [{ data: orders }] = useFetchTwelveOrdersQuery();
+
+  useEffect(() => setAppState(app?.getState()), [app]);
+
+  useEffect(() => {
+    appState?.domain && appState?.token && fetch(
+        "/api/configuration",
+        { headers: [
+          [SALEOR_DOMAIN_HEADER, appState?.domain],
+          ["authorization-bearer", appState?.token!],
```

Comment:
Since we check if `domain` and `token` exist, should `appState.domain` and `appState.token` suffice?